02-insertion-sort/insert_sort.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE HTML>
+<html>
+  <head>
+    <meta content="text/html; charset=utf-8">
+    <title>Insertion-sort</title>
+  </head>
+
+  <body>
+    <h1>Insertion sort</h1>
+    <div id="first_array">
+      <p>Number of elements generated in the array</p>
+      <p><input id="input_size" type = "text" size="10" value=""></p>
+      <p><input id="generate_but" type="button" value="Generate"></p>
+      <p><input id="show_array" type="text" value=""></p>
+    </div>
+    
+    <div id="sort_array">
+        <p><input id="sort_but" type="button" value="Sort"></p>
+        <p><input id="result" type="text" value=""></p>
+    </div>
+    <script src="./scripts/insert_sort.js"></script>
+  </body>
+</html>
```

Comment:
I'm not sure why, but github wants you to add newlines at the end of html files.